### ISSUE TITLE
Add date column to fullgo_version table to track DB load date

### ISSUE THREAD

**Issue by dustine32** [pantherdb/fullgo_paint_update#47]
To help PANTHER and PAINT users clearly see when the full GO update load occurs (thus changing their results) we should display a "loaded on" or "loaded into PANTHER on" date.

pantherdb.org/data/:
![image](https://user-images.githubusercontent.com/2678599/81871456-aeae9c80-952c-11ea-8eb9-a963d23ae25c.png)

This ticket will be for adding the date field to the `fullgo_version` tables in both Panther_XX.X and Curation DBs, along with whatever update coding is necessary to populate this field at update time.

**Comment by pgaudet** [comment]
"GO database version 1.2" seems wrong ? GO releases only go by date, dont they ?

**Comment by dustine32** [comment]
@pgaudet Right! We're going to drop this "1.2" according to the discussion in https://github.com/pantherdb/features-bugs/issues/11 since none of including @mugitty and @huaiyumi can recall its origin.
